tests/aio-create.sh

```diff
@@ -145,6 +145,7 @@ if [ "${RE_JOB_SCENARIO}" == "ceph" ]; then
   git clone https://github.com/rcbops/rpc-ceph /opt/rpc-ceph
   export RPC_MAAS_DIR="$(pwd)"
   pushd /opt/rpc-ceph
+    apt-get install -y apt-transport-https
```

Comment:
Hey @tonytan4ever  It's not clear to me why this package is only installed on ceph/newton -- can you add some details to commit message to clarify?

Reply:
This is because in newton we will encounter this error if this is not installed:

```
PLAY [Deploy Grafana] **********************************************************

TASK [Ensure https repos function] *********************************************
Tuesday 22 May 2018  19:15:33 +0000 (0:00:00.226)       0:16:14.559 *********** 
ok: [aio1]

TASK [Add grafana apt-keys] ****************************************************
Tuesday 22 May 2018  19:15:35 +0000 (0:00:02.031)       0:16:16.591 *********** 
changed: [aio1]

TASK [Add grafana repo] ********************************************************
Tuesday 22 May 2018  19:15:37 +0000 (0:00:02.429)       0:16:19.020 *********** 
changed: [aio1]

TASK [Update Apt cache] ********************************************************
Tuesday 22 May 2018  19:15:38 +0000 (0:00:00.802)       0:16:19.823 *********** 
FAILED - RETRYING: TASK: Update Apt cache (5 retries left).
FAILED - RETRYING: TASK: Update Apt cache (4 retries left).
FAILED - RETRYING: TASK: Update Apt cache (3 retries left).
FAILED - RETRYING: TASK: Update Apt cache (2 retries left).
FAILED - RETRYING: TASK: Update Apt cache (1 retries left).
fatal: [aio1]: FAILED! => {"changed": false, "failed": true, "module_stderr": "", "module_stdout": "", "msg": "MODULE FAILURE"}
```

and this in ceph:
```
TASK [Update Apt cache] ********************************************************
Tuesday 22 May 2018  04:02:00 +0000 (0:00:01.211)       0:19:41.006 *********** 
FAILED - RETRYING: Update Apt cache (5 retries left).
changed: [mon1]
changed: [mon2]
changed: [osd2]
changed: [osd1]
FAILED - RETRYING: Update Apt cache (4 retries left).
changed: [rsyslog1]
changed: [allsvc]
changed: [rgw1]
FAILED - RETRYING: Update Apt cache (3 retries left).
FAILED - RETRYING: Update Apt cache (2 retries left).
FAILED - RETRYING: Update Apt cache (1 retries left).
fatal: [localhost]: FAILED! => {"attempts": 5, "changed": false, "failed": true, "msg": "Failed to update apt cache."}
```
(This is when installing telegraf)

@andymcc's advice aded apt-transport-https to fix them

Reply:
@tonytan4ever thanks! So what happens if we deploy this code for a customer on newton? This is only getting installed in `tests/aio-create.sh` but the listed failure appears to be an installation-related task.

Reply:
@tonytan4ever ok, tracked down @andymcc's commit message in rpc-ceph and think I understand this better now. Perhaps it'd be better to just ensure this package is installed across the board [here](https://github.com/rcbops/rpc-maas/blob/master/tests/aio-create.sh#L137)?